shark_turbine/kernel/wave/iree_utils.py

```diff
@@ -0,0 +1,54 @@
+import torch
+from .utils import compile_and_invoke
+
+
+def get_mmt_asm(lhs_type: str, rhs_type: str, acc_type: str) -> str:
+    acc_dtype = acc_type.split("x")[-1]
```

Comment:
[No action for now]: I think in the future we can also update this to use jinja templates S.T we can create separate files for MLIR templates.

Reply:
I think we can already do
```python
matmul_function = f"""
func.func @mmt(%lhs: tensor<{lhs_type}>, %rhs: tensor<{rhs_type}>) 
...
"""

Reply:
Good point on the jinja templates. Thanks for the pointer of the f-triple string, will change to use that.

Reply:
Ooo, if that is possible, that is also very nice haha

Reply:
+ you don't need `\n` in multiline literals

Reply:
thanks you learn something new everyday :)